apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -169,4 +171,12 @@ export class FileRecord extends BaseEntity {
 	unmarkForDelete(): void {
 		this.deletedSince = undefined;
 	}
+
+	setName(name: string): void {
+		if (name.length === 0) {
+			throw new InternalServerErrorException(ErrorType.FILE_NAME_EMPTY);
```

Comment:
InternalServerErrorException ist ein 500er Fehler. Die 5xx Fehler sind diejenigen auf die alerts in Grafana gesetzt werden (können) um Devops zu informieren wenn zu viele auftreten im Kontext auf es gibt Code Fehler (etwas ist falsch implementiert), es gibt Kubernetes Fehler (Konfig zum S3 ist falsch, Pod ist ausgefallen etc..). 

Ein 4xxer der wäre hier besser, da es falsche Nutzereingabe ist. Höchstwahrscheinlich müsste das auch bereits als ValidationError beim Controller abgewickelt werden, allerdings können natürlich auch über andere Kanäle Services aufgerufen werden, daher sollte hier ein Fehler fliegen. 

Möglichst aber kein InternalServerErrorException und auch kein new Error. Beides ist dann ein 500er.